src/ripple/app/tx/apply.h

```diff
@@ -39,11 +41,24 @@ class Application;
 
     @return The TER code (a `tem` or tesSUCCESS)
 */
-TER
+PreflightResult
```

Comment:
Good catch. That was not my intention.
Option 1: Make the `PreflightResult` and `PreclaimResult` structs public, and make the context member of the struct private.
Option 2: Take `preflight, preclaim, and doApply` out of the public API, and put them under `impl` as well.

I'm leaning toward Option 2 because I've got the transaction queue under `tx/impl` right now, so there is no need for those functions to be public.